pylint/lint.py

```diff
@@ -976,6 +981,7 @@ def run(self, args):
             'load_plugins',
             'ignore',
             'ignore_patterns',
+            'version',
         )
```

Comment:
Just parsing the version arg will do the print and exit, because of the argparse `'version'` action. Note that the `'version'` action does remove all new lines from the string, so it won't be exactly the same as the module version. Will that be a problem?

Reply:
Nope I think this is fine as is!